q2_dada2/assets/run_dada_paired.R

```diff
@@ -199,15 +209,51 @@ errR <- suppressWarnings(learnErrors(filtsR, nreads=nreads.learn, multithread=mu
 ### PROCESS ALL SAMPLES ###
 # Loop over rest in streaming fashion with learned error rates
 denoisedF <- rep(0, length(filtsF))
+ddsF <- vector("list", length(filtsF))
+ddsR <- vector("list", length(filtsF))
 mergers <- vector("list", length(filtsF))
-cat("3) Denoise remaining samples ")
+cat("3) Denoise samples ")
+
+for(j in seq(length(filtsF))) {
+  drpF <- derepFastq(filtsF[[j]])
+  ddsF[[j]] <- dada(drpF, err=errF, multithread=multithread, verbose=FALSE)
+  drpR <- derepFastq(filtsR[[j]])
+  ddsR[[j]] <- dada(drpR, err=errR, multithread=multithread, verbose=FALSE)
+  cat(".")
+}
+cat("\n")
+if(poolMethod == "pseudo") {
+  cat("  Pseudo-pool step ")
+  ### TEMPORARY, to be removed once 1.12 makes its way to Q2
```

Comment:
Incidentally, it looks like bioconda has 1.14 available at this point, we could update the recipe to pin at that version if it was helpful.

Reply:
1.14 will allow `denoise-pacbio` to be added, but I wouldn't want to bump the version before I sort out the `denoise-pyro` discrepancy above.